src/components/header/header.js

```diff
@@ -23,15 +24,40 @@ export default class Header {
       }
       header = createElementsFromString(usersHeader(headersProps));
     }
-    const screenWithHeader = target.appendChild(header);
+    if (target.firstChild) {
```

Comment:
As i mentioned before we can use 'replaceFirstChild' without check on target.firstChild